plugins/uefi/uefi.conf

```diff
@@ -7,3 +7,6 @@ RequireShimForSecureBoot=true
 # the EFI system partition path used
 # if this is is not /boot/efi, /boot, or /efi
 #OverrideESPMountPoint=
+
+# amount of free space required on the ESP, typically the largest flash part
+RequireESPFreeSpace=0x2000000
```

Comment:
It might be good to switch to uefi.conf.in and fill this in from meson so it doesn't need to be hard coded in two places

Reply:
Is `meson.build` the right place to set this? Seems kind a magic. Why don't we set the default to 0 in the config file, and then fall back to the default in `fu-uefi-common.h`?

Reply:
Sure, or unset it in the conf file, either way is fine.

Reply:
> or unset it in the conf file

Right, that's way more obvious. I've done that, thanks.